### ISSUE TITLE
[A11Y[Bug][Android] Checkbox: duplicate role and status

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#360]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S22

### OS Version

14

### App version

Android 0.6.0

### Describe the issue

Checkbox is heard as a tickbox and a checkbox by Talkback, so we hear twice the element as seen below:

<img width="300" height="60" alt="" src="https://github.com/user-attachments/assets/4098bebd-fac7-45ca-9cfa-2ede403e002d" />

<img width="297" height="56" alt="" src="https://github.com/user-attachments/assets/0b311595-ccef-40d4-8402-599808e330e7" />

### Expected Behavior

Remove tickbox role

**Comment by pya35** [comment]
linked to #47

**Comment by nouha06** [comment]
@pya35 Could you provide me with the build version? I’m unable to reproduce this issue. I’ve tested different versions of 0.6.0 and I always hear only “Not checked, Checkbox”; I haven’t encountered “tick box”.

![Image](https://github.com/user-attachments/assets/16e23ba1-4013-49c2-bba8-7e4857d5ebe6)

**Comment by nouha06** [comment]
Hi @pya35 ,
Just following up on this ticket. Have you had a chance to look at it?

**Comment by pya35** [comment]
hi @nouha06, I will make a deep check on Flutter apps on next week

**Comment by pya35** [comment]
not reproduced on beta 0.6.0 (469)

**Comment by B3nz01d** [comment]
Closed during Sprint # 23
